### DISCUSSION TITLE
Can CustomJSforFx also be applied to Firefox Android [root]?

### DISCUSSION THREAD

**Discussion by LeeBinder** [Aris-t2/CustomJSforFx#66] [Q&A] [answered]
And if so, where would the files need to be copied to? Would the Firefox (or Fennec, Iceraven etc.) apk itself need to be modified (and resigned), too? Or is that not possible in general at this project's current state?

Needless to say, prerequisite is a rooted system.

**Comment by Aris-t2** [comment, marked as answer]
Unfortunately the kind of customization offered by this project is for desktop systems only.

**Comment by LeeBinder** [comment]
Thanks Aris for the clarification.
